tests/osaka/eip7883_modexp_gas_increase/conftest.py

```diff
@@ -0,0 +1,125 @@
+"""Shared pytest definitions local to EIP-7883 tests."""
+
+from typing import Dict
+
+import pytest
+
+from ethereum_test_tools import (
+    EOA,
+    Account,
+    Address,
+    Alloc,
+    Environment,
+    Storage,
+    Transaction,
+)
+from ethereum_test_tools.vm.opcode import Opcodes as Op
+
+from .spec import Spec
+
+
+@pytest.fixture
+def env() -> Environment:
+    """Environment fixture."""
+    return Environment()
+
+
+@pytest.fixture
+def sender(pre: Alloc):
+    """Create and fund an EOA to be used as the transaction sender."""
+    return pre.fund_eoa()
+
+
+@pytest.fixture
+def gas_measure_contract(pre: Alloc):
+    """Deploys a simple contract to call ModExp and store its success."""
+    # TODO: fix the gas accounting
```

Comment:
The tricky part is where to insert the `Op.GAS` since you also want to store the call return, so you could do `Op.GAS + call + Op.GAS` and then use swaps to move the stack, subtract the gas measurements, and save into storage.
You can use the traces to see exactly where each value is in the stack when the code runs.